src/histolab/tiler.py

```diff
@@ -690,47 +725,45 @@ def extract(self, slide: Slide, report_path: str = None):
 
     # ------- implementation helpers -------
 
-    def _highest_score_tiles(self, slide: Slide) -> List[Tuple[float, CoordinatePair]]:
+    def _highest_score_tiles(
+        self, slide: Slide, scaled: bool = False
+    ) -> List[Tuple[float, CoordinatePair]]:
         """Calculate the tiles with the highest scores and their extraction coordinates.
 
         Parameters
         ----------
         slide : Slide
             The slide to extract the tiles from.
+        scaled: bool
+            Whether to return the scores scaled between 0 and 1.
 
         Returns
         -------
         List[Tuple[float, CoordinatePair]]
-            List of tuples containing the score and the extraction coordinates for the
-            tiles with the highest score. Each tuple represents a tile.
-        List[Tuple[float, CoordinatePair]]
-            List of tuples containing the scaled score between 0 and 1 and the
-            extraction coordinates for the tiles with the highest score. Each tuple
-            represents a tile.
+            List of tuples containing the (scaled) scores and the extraction coordinates
+            for the tiles with the highest scores. Each tuple represents a tile.
```

Comment:
can you be more explicit in this part of the docstring? the `(scaled)` is not clear to me to understand that the param can be the discriminant to scale or not the scores